source/Application/Controller/Admin/OrderMain.php

```diff
@@ -166,7 +166,7 @@ public function sendOrder()
             $oOrder->save();
 
             // #1071C
-            $oOrderArticles = $oOrder->getOrderArticles(true);
+            $oOrder->getOrderArticles(true);
```

Comment:
I'm not sure about this one so I only cleared the variable assignment. The called method seems to execute something relevant which is relevant for the current method.